plugins/CoreHome/javascripts/broadcast.js

```diff
@@ -804,8 +804,8 @@ var broadcast = {
             var value = url.substring(startPos + lookFor.length, endStr);
 
             // we sanitize values to add a protection layer against XSS
-            // &segment= value is not sanitized, since segments are designed to accept any user input
-            if(param != 'segment') {
+            // &segment= (and &popover=) value is not sanitized, since segments are designed to accept any user input
+            if(param != 'segment' && param != 'popover') {
```

Comment:
This change was required in order to fix a UI test with XSS values. The popover param made problems for row evolution with labels containing special chars. Closing the popover didn't remove the param from url without that change.